FEniCS CI is broken as the conda package doesn't work anymore (again!):
```py
/usr/share/miniconda/envs/fenicsproject/bin/python
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/usr/share/miniconda/envs/fenicsproject/lib/python3.10/site-packages/dolfin/__init__.py", line 34, in <module>
    from .cpp import __version__
ImportError: libsuperlu_dist.so.7: cannot open shared object file: No such file or directory
```
The solution from https://fenicsproject.discourse.group/t/installed-but-looking-for-libsuperlu-dist-so-6/7016 doesn't work anymore.
It might be related to a Python version as a recent job [here ](https://github.com/IvanYashchuk/fecr/actions/runs/3082168111/jobs/4981534213) passes, but uses Python 3.8

--- a/.github/workflows/ci-fenics.yml
+++ b/.github/workflows/ci-fenics.yml
@@ -36,7 +36,7 @@ jobs:
         run: |
           conda config --set always_yes yes
           conda config --add channels conda-forge
-          mamba create -n fenicsproject -c conda-forge fenics superlu_dist=6.2.0
+          mamba create -n fenicsproject -c conda-forge fenics
           conda activate fenicsproject
           which python
           python -c "from dolfin import *"

